stories/assets/scss/_variables.scss

```diff
@@ -39,6 +39,7 @@ $color-dark-azure : #00C1FF;
 // FONT SIZE
 $font-size-20: 16px;
 $font-size-14: 14px;
+$font-size-16: 20px;
```

Comment:
@Shubham-Sharma-77 can you change the variable name 16px should be $font-size-16 and 20px should $font-size-20